doc/RelNotes.md

```diff
@@ -25,6 +25,7 @@ if you find any other issue with this release.
 * Many classes of the Server part have been renamed
 * Various API fixes
 * Project-wide codestyle cleanup
+* DesktopStreamer: Only allow desktop interaction on supported platforms (OSX)
```

Comment:
The functionality was introduced in v0.5 so that would not work, but I changed the comment to:
DesktopStreamer: Minor fixes for desktop interaction, only show checkbox on supported platforms (OSX)